docs/user-guide/api-mediation/using-multi-factor-authentication.md

```diff
@@ -0,0 +1,42 @@
+# Using multi-factor authentication (MFA)
+
+Zowe offers the option to use multi-factor authentication (MFA) systems, which require users to provide multiple authentication factors during logon to verify the user's identity. When using multi-factor authentication, it is necessary that each authentication factor be from a separate category of credential types. While multi-factor authentication is supported by Zowe, there are limitations for this feature to function properly. This topic explains the limitations of using MFA in Zowe and recommendations to address these limitations.
+
+Beginning with Zowe version 1.8.0, the Zowe App Framework, Desktop, and all apps present in the SMP/E or convenience builds support out-of-band MFA. Users are required to enter an MFA assigned token or passcode into the password field of the Desktop login screen. Alternatively, users can access the app-server `/auth` REST API endpoint.
```

Comment:
We can remove the Beginning with Zowe version 1.8.0 and start with: 

The Zowe API Mediation Layer, Zowe App Framework, and all apps present ... band MFA. Users are required to enter an MFA assigned token or passcode into the password field of the Desktop login screen or authentication to the API Catalog. 

Alternatively user can access one of the authentication endpoints e.g. `/gateway/auth/login` within the API Mediation Layer or via App-servers `/auth` REST API endpoint.